server/api/src/delayed_init.py

```diff
@@ -0,0 +1,53 @@
+import asyncio
+import aiofiles
+import aiohttp
+import os
+
+from utils import parse_yagna_key
+from time import time
+
+async def is_yagna_available(app):
+    global YAGNA_ACCOUNT, YAGNA_APP_KEY
```

Comment:
Why not make these into `app` attributes like `yagna_available`? Maybe `yagna_available` won't even be needed anymore if we can check these two?

Reply:
I not yet sure whether I would need them (or APPKEY) have in env vars so I will postpone the change to the next, further iteration